src/index.js

```diff
@@ -1 +1,76 @@
+/* eslint-disable no-console */
+// /* eslint-disable no-console */
 'use strict';
+
+import http from 'node:http';
+import formidable, {errors as formidableErrors} from 'formidable';
+import fs from 'fs';
+import path from 'node:path';
+import zlib from 'zlib';
+
+const server = http.createServer(async (req, res) => {
+  const url = new URL(req.url, `http://${req.headers.host}`);
+  // const fileName = url.pathname.slice(1) || 'index.html';
+  // const filePath = path.resolve('public', fileName);
+
+  if (req.url === '/download' && req.method.toLowerCase() === 'post') {
+    // parse a file upload
+    const form = formidable({});
+    let fields;
+    let files;
+    try {
+      [fields, files] = await form.parse(req);
+    } catch (err) {
+      // example to check for a very specific error
```

Comment:
Try to write obvious code with meaningful variable and function naming (without comments)